main.go

```diff
@@ -518,29 +526,35 @@ func postAssignee(c *gin.Context, db *sql.DB) {
 		return
 	}
 
-	for _, assigneeUsername := range assignee.Usernames {
-		assigneeUser, err := GetUserByUserName(db, assigneeUsername)
-		if err != nil {
-			fmt.Println(err)
-			c.AbortWithStatus(http.StatusInternalServerError)
-			return
-		}
+	if authorizedUserId == currentRepo.UserId {
```

Comment:
Instead of `if authorizedUserId == currentRepo.UserId` do `if authorizedUserId != currentRepo.UserId` and you can do early return.

Reply:
done